yacht/make_training_data_from_sketches.py

```diff
@@ -38,27 +80,34 @@ def main(args):
 
     # make sure reference database file exist and valid
     logger.info("Checking reference database file")
-    if os.path.splitext(ref_file)[1] != '.zip':
+    if os.path.splitext(ref_file)[1] != ".zip":
         raise ValueError(
-            f"Reference database file {ref_file} is not a zip file. Please a Sourmash signature database file with Zipfile format.")
-    utils.check_file_existence(str(Path(ref_file).absolute()),
-                               f'Reference database zip file {ref_file} does not exist.')
+            f"Reference database file {ref_file} is not a zip file. Please a Sourmash signature database file with Zipfile format."
+        )
+    utils.check_file_existence(
+        str(Path(ref_file).absolute()),
+        f"Reference database zip file {ref_file} does not exist.",
+    )
 
     # Create a temporary directory with time info as label
     logger.info("Creating a temporary directory")
-    path_to_temp_dir = os.path.join(outdir, prefix + '_intermediate_files')
+    path_to_temp_dir = os.path.join(outdir, prefix + "_intermediate_files")
     if os.path.exists(path_to_temp_dir) and not force:
-        raise ValueError(f"Temporary directory {path_to_temp_dir} already exists. Please remove it, use '--force', or given a new prefix name using parameter '--prefix'.")
+        raise ValueError(
+            f"Temporary directory {path_to_temp_dir} already exists. Please remove it, use '--force', or given a new prefix name using parameter '--prefix'."
```

Comment:
And then this line that goes over the 79 characters...

Reply:
as for me, it still improves readability a bit, but I agree it is better to divide this line.